charts/newrelic-infra-operator/templates/admission-webhooks/job-patch/clusterrole.yaml

```diff
@@ -2,13 +2,13 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ template "newrelic-infra-operator.fullname" . }}-admission
+  name: {{ include "newrelic.common.naming.fullname" . }}-admission
```

Comment:
there is a new feature added by @sigilioso  to manage the name suffix in version 1.0.2 of the common library. you could have a look at https://github.com/newrelic/k8s-metadata-injection/pull/163 to check how was used. I think this applies also here

Reply:
I agree, [this is the commit where it's used](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/be9ffd545b6b70cc0a33cbbf3eaa6eff719c4337), please ping me if you have any question.

Reply:
Fixed. Thanks!